service/src/main/resources/application.yml

```diff
@@ -92,7 +92,7 @@ profile:
     managed-app-tenant-id: ${env.azure.managedAppTenantId}
 
     application-offers:
-      - name: terra-workspace-dev
+      - name: aherbst-202202024-preview
```

Comment:
@aherbst-broad is it OK to check this in?

Reply:
@aherbst-broad I'm assuming **no** given https://github.com/broadinstitute/terra-helmfile/pull/3312/files?